spec/controllers/my_account_controller_spec.rb

```diff
@@ -22,7 +22,7 @@
       context 'when requesting deletion of another user' do
         let(:logged_in_user) { User.excluding(user_to_destroy).first! }
 
-        it 'does not delete the specified user', :no_bullet do
```

Comment:
I'm not sure why I had this `no_bullet` meta tag. I guess it passes with Prosopite active. 🤷‍♂️